python/xorbits/numpy/numpy_adapters/core.py

```diff
@@ -71,6 +72,7 @@ def _get_output_type(func: Callable) -> MarsOutputType:
         return _NO_ANNOTATION_FUNCS.get(func, MarsOutputType.object)
 
 
+@lru_cache(maxsize=1)
 def collect_numpy_module_members(np_mod: ModuleType) -> Dict[str, Any]:
```

Comment:
This method is already called `collect_numpy_module_members`, do we still need to pass the numpy module into it?

Reply:
hi, thanks for your review! I found that currently, we are passing different numpy modules (```numpy```, ```numpy.fft```, ```numpy.linalg``` etc) to this function to generate the fallback methods in their init.py. Should I modify this logic? Thanks.

Reply:
Oh, it's OK when passing submodules, but just curious that the functions for numpy & pandas act differently.